src/styles/blocks/page.scss

```diff
@@ -0,0 +1,13 @@
+.page {
+  font-family: "Inter", sans-serif;
+  font-weight: normal;
+  font-size: 16px;
+  color: $colorBlack;
+
+  scroll-behavior: smooth;
```

Comment:
The `scroll-behavior` property should be applied to the `html` or `body` element. Move it to the `&__body` selector. 📜